cargo-pgx/src/command/install.rs

```diff
@@ -287,8 +286,14 @@ pub(crate) fn build_extension(
     let command = command.stderr(Stdio::inherit());
     let command_str = format!("{:?}", command);
     println!(
-        "building extension with features `{}`\n{}",
-        features_arg, command_str
+        "{} extension with features {}",
+        "    Building".bold().green(),
+        features_arg.cyan()
+    );
+    println!(
+        "{} command {}",
+        "     Running".bold().green(),
+        command_str.cyan()
```

Comment:
This is only to make things look a little nicer.

Reply:
I love it!

Reply:
❤️